mne/decoding/transformer.py

```diff
@@ -248,6 +248,12 @@ def inverse_transform(self, X, y=None):
 class PSDEstimator(TransformerMixin):
     """Compute power spectrum density (PSD) using a multi-taper method
 
+    This structures data so that it can be easily incorporated into
+    scikit-learn pipelines. It relies heavily on the _psd_multitaper
```

Comment:
don't refer to private function ... I think you don't even need any of this (L251 - 255) anymore because we have a manual page on decoding + you refactored the functions which clarifies everything.